src/scss/includes/panels/poi_panel.scss

```diff
@@ -16,6 +18,31 @@ $BLOCK_INDENT: 60px;
   display: none;
 }
 
+.poi_panel__content__card {
+  display: none;
+}
+
+.poi_panel__content__card__content {
+  float: left;
+  width: calc(100vw - (16px + 130px + 8px));
```

Comment:
ouch  :confused: , please use constants here
Where do these values come from ? And how are they related to other changes ?